### PR TITLE
chore: update Angular deps and bump to 4.2

### PR DESCRIPTION
**Pull request by sis0k0**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
     },
     "name": "tns-template-hello-world-ng",
     "main": "main.js",
-    "version": "3.1.1",
+    "version": "4.2.0",
```
Comment (vchimev) [inline package.json]: Are we sure about this? I thought we'd like to follow the Angular's version only for the `nativescript-angular` plugin ...

Comment (sis0k0) [inline package.json]: @dtopuzov @vakrilov what do you think?

File: package.json [hunk 2]
```diff
@@ -31,20 +31,20 @@
         "nativescript-theme-core": "~1.0.2",
         "nativescript-angular": "~3.1.0",
```
Comment (vchimev) [inline package.json]: Yeah, we've have been updated the version of `nativescript-angular` as well. :)

Comment (vchimev) [inline package.json]: Yeah, we could have been updated the version of `nativescript-angular` as well. 😸

File: package.json (hunk 1 above)
Comment (vchimev) [inline package.json]: @rosen-vladimirov should have an opinion on this. :)

**Pull request merged**